recipes/date/all/conanfile.py

```diff
@@ -1,29 +1,91 @@
 import os
-from conans import ConanFile, tools
+from conans import ConanFile, CMake, tools
 
 
 class DateConan(ConanFile):
     name = "date"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/HowardHinnant/date"
     description = "A date and time library based on the C++11/14/17 <chrono> header"
-    topics = ("date", "datetime", "timezone", "calendar", "time", "iana-database")
+    topics = ("date", "datetime", "timezone",
+              "calendar", "time", "iana-database")
     license = "MIT"
-    exports_sources = ["patches/*"]
-    no_copy_source = True
+    exports_sources = ["patches/*", "CMakeLists.txt"]
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package"
```

Comment:
do we want to remove this cmake_find_package and get rid of find Threads from roiginal cmake?

Reply:
`cmake_find_package` is useless because `cmake.patch` removed `find_package( CURL REQUIRED )`
`find_package( Threads )` is necessary in original CMakeLists.